### PR TITLE
[SPARK-48963][SPARK-34553] Add instructions for JIRA_ACCESS_TOKEN and GITHUB_OAUTH_KEY in release guide

### PR DESCRIPTION
**Pull request by yaooqinn**

### CODE REVIEW

**Review by HyukjinKwon — APPROVED** [review]

File: release-process.md [hunk 1]
```diff
@@ -401,16 +401,18 @@ $ export RELEASE_TAG=v1.1.1
 $ export PREVIOUS_RELEASE_TAG=v1.1.0
 # Generate initial contributors list, likely with warnings
 $ ./generate-contributors.py
-# set JIRA_USERNAME, JIRA_PASSWORD, and GITHUB_API_TOKEN
+# set GITHUB_OAUTH_KEY.
```
Comment (viirya) [inline release-process.md]:
```suggestion
# Set GITHUB_OAUTH_KEY.
```

File: release-process.md [hunk 2]
```diff
@@ -401,16 +401,18 @@ $ export RELEASE_TAG=v1.1.1
...
+$ export GITHUB_OAUTH_KEY=blabla
+# set either JIRA_ACCESS_TOKEN (for 4.0.0 and later) or JIRA_USERNAME / JIRA_PASSWORD.
```
Comment (viirya) [inline release-process.md]:
```suggestion
# Set either JIRA_ACCESS_TOKEN (for 4.0.0 and later) or JIRA_USERNAME / JIRA_PASSWORD.
```

**Review by viirya — APPROVED** [review]

File: site/release-process.html [hunk 3]
```diff
@@ -541,16 +541,18 @@ <h4 id="update-the-rest-of-the-spark-website">Update the rest of the Spark websi
 $ export PREVIOUS_RELEASE_TAG=v1.1.0
 # Generate initial contributors list, likely with warnings
 $ ./generate-contributors.py
-# set JIRA_USERNAME, JIRA_PASSWORD, and GITHUB_API_TOKEN
+# set GITHUB_OAUTH_KEY.
```
Comment (yaooqinn) [inline site/release-process.html]:
```suggestion
# Set GITHUB_OAUTH_KEY.
```

File: site/release-process.html [hunk 4]
```diff
@@ -541,16 +541,18 @@ <h4 id="update-the-rest-of-the-spark-website">Update the rest of the Spark websi
 $ export PREVIOUS_RELEASE_TAG=v1.1.0
 # Generate initial contributors list, likely with warnings
 $ ./generate-contributors.py
-# set JIRA_USERNAME, JIRA_PASSWORD, and GITHUB_API_TOKEN
+# Set GITHUB_OAUTH_KEY.
+$ export GITHUB_OAUTH_KEY=blabla
+# set either JIRA_ACCESS_TOKEN (for 4.0.0 and later) or JIRA_USERNAME / JIRA_PASSWORD.
```
Comment (yaooqinn) [inline site/release-process.html]:
```suggestion
# Set either JIRA_ACCESS_TOKEN (for 4.0.0 and later) or JIRA_USERNAME / JIRA_PASSWORD.
```

**Comment by yaooqinn** [comment]
Merged to asf-site. Thank you @HyukjinKwon @viirya

**Review by dongjoon-hyun — COMMENTED** [review]
+1, late LGTM.

**Pull request merged**